### PR TITLE
[INLONG-8394][Manager] TenantRole list interface support fuzzy match

### LINKED ISSUE #8394: [Improve][Manager] TenantRole list interface support fuzzy match
### Description

Current list interface of TenantUserRole only support fully match of tenant. 
Now make it support fuzzy match to improve the ability of this interface

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Fixes #8394 

### Motivation

To enhance the tenant role list interface, make it support fuzzy match

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/user/TenantRolePageRequest.java [hunk 1]
```diff
@@ -38,8 +38,8 @@ public class TenantRolePageRequest extends PageRequest {
     @ApiModelProperty(value = "Role code")
     private String roleCode;
 
-    @ApiModelProperty(value = "Tenant")
-    private String tenant;
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/user/TenantRolePageRequest.java:42]: Why delete it？

File: inlong-manager/manager-dao/src/main/resources/mappers/TenantUserRoleEntityMapper.xml [hunk 2]
```diff
@@ -64,8 +64,10 @@
         <if test="roleCode != null and roleCode != ''">
             and role_code = #{roleCode,jdbcType=VARCHAR}
         </if>
-        <if test="tenant != null and tenant != ''">
-            and tenant = #{tenant, jdbcType=VARCHAR}
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/resources/mappers/TenantUserRoleEntityMapper.xml:68]: Why delete it？

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/user/TenantRolePageRequest.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/user/TenantRolePageRequest.java:42]: This interface should not support multiple tenant,  **_keyword_** is use to fuzzy match the tenant.

File: inlong-manager/manager-dao/src/main/resources/mappers/TenantUserRoleEntityMapper.xml (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/resources/mappers/TenantUserRoleEntityMapper.xml:68]: ditto

**Review by fuweng11 — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**